libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/RustMatrixClient.kt

```diff
@@ -87,7 +87,7 @@ class RustMatrixClient constructor(
 
     override val sessionId: UserId = UserId(client.userId())
     private val app = client.app().use { builder ->
-        builder.finish()
+        builder.withEncryptionSync(false, "").finish()
```

Comment:
Should be part of another PR if we really enable this I think

Reply:
My bad, I thought I had removed that!